modules/bmo-config-using-bare-metal-operator.adoc

```diff
@@ -1,12 +1,21 @@
 // This is included in the following assemblies:
 //
-// post_installation_configuration/post-install-bare-metal-configuration.adoc
+// * installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc
 
-:_module-type: CONCEPT
-[id="bmo-about-the-bare-metal-operator_{context}"]
-= About the Bare Metal Operator
+:_mod-docs-content-type: CONCEPT
+[id="bmo-config-using-bare-metal-operator_{context}"]
+= Configuration using the Bare Metal Operator
 
-Use the Bare Metal Operator (BMO) to provision, manage, and inspect bare-metal hosts in your cluster.
+When deploying {product-title} on bare-metal hosts, there are times when you need to make changes to the host either before or after provisioning. This can include inspecting the host's hardware, firmware, and firmware details. It can also include formatting disks or changing modifiable firmware settings.
+
+You can use the Bare Metal Operator (BMO) to provision, manage, and inspect bare-metal hosts in your cluster. The BMO can complete the following operations:
+
+* Provision bare-metal hosts to the cluster with a specific image.
+* Turn on or off a host.
+* Inspects hardware details of the host and report them to the bare metal host.
```

Comment:
Change "Inspects" to "Inspect" for parallelism. Also, hyphenate "bare metal" before "host".